README.md

```diff
@@ -0,0 +1,77 @@
+# SkyPortal Mobile
+
+This is the mobile version of the [SkyPortal](https://github.com/skyportal/skyportal) platform. It is built with Ionic
+React and is available for both iOS and Android.
+
+## Development requirements and setup
+
+### Node.js and npm
+
+To run the app locally, you will need to have Node.js and npm installed. You can install them
+using [nvm](https://github.com/nvm-sh/nvm) or just install a recent version from the [node website](https://nodejs.org/).
+
+### Android Studio
+
+To be able to build the Android version, you will need to have Android Studio installed with the Android SDK and the
+Android SDK Platform-Tools. You can follow the instructions [here](https://developer.android.com/studio/install) to
+install Android Studio. Then, open Android Studio, go to **Tools -> SDK Manager** and install the latest stable Android
+SDK and Android SDK Platform-Tools as shown below ![img.png](doc/android_studio_sdk_install.png)
+You will also have to set the ANDROID_HOME environment variable to the path of the Android SDK. You can do that by
+adding the following line to your `.bashrc` or `.zshrc` file:
+
+```bash
+export ANDROID_HOME=$HOME/Android/Sdk
```

Comment:
looks incorrect, `Library` should be in there (as per your screenshot). Worth putting a note or something in italic to say that this path could differ based on where the user decided to install the SDK. If they specified a different path, they should use that here.